tests/sparseml/pytorch/optim/test_modifier_quantization.py

```diff
@@ -233,6 +235,7 @@ def test_quantization_modifier_yaml():
         quantize_embeddings=quantize_embeddings,
         reduce_range=reduce_range,
         quantize_linear_activations=quantize_linear_activations,
+        num_calibration_steps=num_calibration_steps,
```

Comment:
need to add asserts here for `num_calibration_steps` (follow logic below)

Reply:
Added